Dockerfile

```diff
@@ -1,17 +1,29 @@
-FROM oamdev/docker-terraform-base:1.0.9
+FROM alpine:3.13
+
+RUN \
+  apk update && \
+  apk add bash py-pip && \
+  apk add --virtual=build gcc libffi-dev musl-dev openssl-dev python3-dev make && \
+  apk add curl jq python3 ca-certificates git openssl unzip wget && \
+  pip --no-cache-dir install -U pip && \
+  pip install azure-cli && \
```

Comment:
Why used the azure-cli here?

Reply:
When the provide is `azurerm`, it's needed.